app/parsers/bulkrax/csv_parser.rb

```diff
@@ -91,8 +91,10 @@ def required_elements?(record)
     def missing_elements(record)
       keys_from_record = keys_without_numbers(record.reject { |_, v| v.blank? }.keys.compact.uniq.map(&:to_s))
       keys = []
-      importerexporter.mapping.map do |k, v|
-        v[:from].each do |vf|
+      # Because we're persisting the mapping in the database, these are likely string keys.
+      # However, there's no guarantee.  So, we need to ensure that by running stringify.
+      importerexporter.mapping.stringify_keys.map do |k, v|
+        Array.wrap(v['from'])).each do |vf|
```

Comment:
```suggestion
        Array.wrap(v['from']).each do |vf|
```

should fix the syntax error